<ui version="4.0" >
 <class>toNewConnectionUI</class>
 <widget class="QDialog" name="toNewConnectionUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>New connection</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>11</number>
   </property>
   <property name="topMargin" >
    <number>11</number>
   </property>
   <property name="rightMargin" >
    <number>11</number>
   </property>
   <property name="bottomMargin" >
    <number>11</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="13" column="3" >
    <widget class="QPushButton" name="CancelButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2" >
    <widget class="QPushButton" name="OkButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Ok</string>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="12" row="1" column="0" >
    <widget class="toListView" native="1" name="Previous" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="TextLabel2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>A list of previous connections. Select to bring over to the right.</string>
     </property>
     <property name="text" >
      <string>&amp;Previous connections</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>Previous</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3" >
    <widget class="QLineEdit" name="Password" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3" >
    <widget class="QLabel" name="TextLabel1" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Username to connect with</string>
     </property>
     <property name="text" >
      <string>&amp;Username</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>Username</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3" >
    <widget class="QLabel" name="TextLabel1_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Password to connect with.</string>
     </property>
     <property name="text" >
      <string>&amp;Password</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>Password</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3" >
    <widget class="QLineEdit" name="Username" />
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QLabel" name="ProviderLabel" >
     <property name="text" >
      <string>Connection provider</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>Provider</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="QComboBox" name="Provider" />
   </item>
   <item row="10" column="1" colspan="3" >
    <widget class="QComboBox" name="Database" >
     <property name="editable" >
      <bool>true</bool>
     </property>
     <property name="autoCompletion" >
      <bool>true</bool>
     </property>
     <property name="duplicatesEnabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3" >
    <widget class="QLabel" name="TextLabel3" >
     <property name="toolTip" >
      <string>Select the database ID to connect to.</string>
     </property>
     <property name="text" >
      <string>&amp;Database</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>Database</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3" >
    <widget class="QComboBox" name="Host" >
     <property name="editable" >
      <bool>true</bool>
     </property>
     <property name="autoCompletion" >
      <bool>true</bool>
     </property>
     <property name="duplicatesEnabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3" >
    <widget class="QLabel" name="HostLabel" >
     <property name="toolTip" >
      <string>Select the database ID to connect to.</string>
     </property>
     <property name="text" >
      <string>&amp;Host</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>Host</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2" >
    <widget class="QLabel" name="PortLabel" >
     <property name="toolTip" >
      <string>Select the database ID to connect to.</string>
     </property>
     <property name="text" >
      <string>&amp;Port</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>Host</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="3" >
    <widget class="QSpinBox" name="Port" >
     <property name="maximum" >
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="3" >
    <widget class="QGroupBox" name="OptionGroup" >
     <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="leftMargin" >
       <number>11</number>
      </property>
      <property name="topMargin" >
       <number>11</number>
      </property>
      <property name="rightMargin" >
       <number>11</number>
      </property>
      <property name="bottomMargin" >
       <number>6</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="12" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>toListView</class>
   <extends>QWidget</extends>
   <header>toresultview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Provider</tabstop>
  <tabstop>Username</tabstop>
  <tabstop>Password</tabstop>
  <tabstop>Host</tabstop>
  <tabstop>Port</tabstop>
  <tabstop>Database</tabstop>
  <tabstop>OptionGroup</tabstop>
  <tabstop>OkButton</tabstop>
  <tabstop>CancelButton</tabstop>
  <tabstop>Previous</tabstop>
 </tabstops>
 <includes>
  <include location="local" >toresultview.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>toNewConnectionUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Provider</sender>
   <signal>activated(int)</signal>
   <receiver>toNewConnectionUI</receiver>
   <slot>changeProvider()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Host</sender>
   <signal>textChanged(QString)</signal>
   <receiver>toNewConnectionUI</receiver>
   <slot>changeHost()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OkButton</sender>
   <signal>clicked()</signal>
   <receiver>toNewConnectionUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>612</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel" >
     <x>374</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
